src/platform/sdl/main.c

```diff
@@ -27,7 +27,9 @@
 
 #include <errno.h>
 #include <signal.h>
+#ifndef _MSC_VER
 #include <sys/time.h>
```

Comment:
It's fine to remove this include entirely. The right one should be in common.h now anyway